spring-integration-file/src/main/java/org/springframework/integration/file/remote/synchronizer/AbstractInboundFileSynchronizingMessageSource.java

```diff
@@ -153,6 +159,34 @@ public void afterPropertiesSet() throws Exception {
 		}
 	}
 
+	@Override
+	public void start() {
+		this.running = true;
+	}
+
+	@Override
+	public void stop() {
+		this.running = false;
+		if (this.synchronizer instanceof Closeable) {
+			try {
+				((Closeable) this.synchronizer).close();
+			}
+			catch (IOException e) {
+				logger.error("Error closing synchronizer", e);
+			}
+		}
+	}
+
+	@Override
+	public boolean isRunning() {
+		return this.running;
+	}
+
+	@Override
+	public String getComponentType() {
+		return "file:synchronizing-message-source";
+	}
```

Comment:
I think this is overhead. Since this class is `abstract` only the target impl should take care about the note about component type. As you see by the `(S)FtpInboundFileSynchronizingMessageSource`, they do that.
So, I'm going to remove this method on merge